.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug Report
+about: Something is not working
+---
+
+<!--
+   Thank you for submitting an issue. Please fill in the template below
+   information about the bug you encountered.
+-->
+
+#### Summary
+<!-- Please explain the bug in a few short sentences -->
+
+#### What Should Happen Instead?
+<!-- Please explain what the expected behavior is -->
+
+#### Reproduction Steps
+<!-- Are you able to consistently reproduce the issue? Please add a list of steps that lead to the bug. -->
+
+1. ...
+2. ...
+
+#### Can you suggest a fix?
+<!-- (This section is optional). How do you propose that the issue be fixed? -->
+
```

Comment:
Let's add a section to ask for basic system info, e.g. `snap version`, `uname -a`, `snap list k8s`, infra (?)

Reply:
Updated. We should exchange that with "Provide the tar file from `k8s inspect`" once this is implemented.